libcloud/compute/drivers/outscale.py

```diff
@@ -7948,3 +7949,283 @@ def _to_key_pair(self, key_pair):
             private_key=private_key,
             fingerprint=key_pair["KeypairFingerprint"],
             driver=self)
+
+    def ex_create_master_key(
```

Comment:
Just drag those methods before the private methods for more compliance with the contributions guidlelines